src/app/CliArgs.cpp

```diff
@@ -67,6 +70,19 @@ CliArgs::parse(int argc, char const* argv[])
         return Action{Action::Exit{EXIT_SUCCESS}};
     }
 
+    if (parsed.count("config-description") != 0u) {
+        std::filesystem::path filePath = parsed["config-description"].as<std::string>();
+        if (!filePath.empty() && !filePath.string().ends_with(".md"))
+            filePath += ".md";
```

Comment:
This is quite unexpected behaviour. I think Clio should use exactly what user passed, without any modiffications.